.github/gargc.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Inspired from https://gist.github.com/ahmetb/7ce6d741bd5baa194a3fac6b1fec8bb7
+
+IFS=$'\n\t'
+set -eou pipefail
+
+if [[ "$#" -ne 2 || "${1}" == '-h' || "${1}" == '--help' ]]; then
+	cat >&2 <<"EOF"
+gargc.sh cleans up tagged or untagged images pushed before specified date
+for a given image
+USAGE:
+  gargc.sh REPOSITORY DATE
+EXAMPLE
+  gargc.sh  europe-docker.pkg.dev/gitpod-artifacts/docker-dev-pr/workspace-base-images-pr 2017-04-01
+  would clean up all images tags pushed before 2017-04-01 for
+  europe-docker.pkg.dev/gitpod-artifacts/docker-dev-pr/workspace-base-images-pr image.
+EOF
+	exit 1
+elif [[ "${#2}" -ne 10 ]]; then
+	echo "wrong DATE format; use YYYY-MM-DD." >&2
+	exit 1
+fi
+
+main() {
+	local C=0
+	IMAGE="${1}"
+	DATE="${2}"
```

Comment:
Maybe it would be better if we can specify `>= 2017-04-01` or `> 20217-04-01`. The reason for this is to make it clearer whether it is included or not. If it could be done easily, it would be wonderful.

Reply:
I have included the information that the deletion is for all layers pushed **before** a certain date. So, the date is not included. I also updated the description to reflect this.